messaging/src/main/java/org/axonframework/eventstreaming/Tag.java

```diff
@@ -32,9 +30,9 @@
  * has been published, typically referring to domain-specifics used for identification.
  * <p>
  * Such a {@code Tag} is typically used by the {@link EventCriteria} as a filter when
- * {@link EventStoreTransaction#source(SourcingCondition, ProcessingContext) sourcing},
- * {@link StreamableEventSource#open(String, StreamingCondition) streaming} or
- * {@link EventStoreTransaction#appendEvent(EventMessage) appending} events.
+ * {@code EventStoreTransaction#source(SourcingCondition, ProcessingContext) sourcing},
```

Comment:
Nit: I think it's better to just omit the links and code references. The fact it's part of the `EventCriteria` is sufficient.